pandas/tests/extension/base/setitem.py

```diff
@@ -338,7 +338,8 @@ def test_setitem_slice_array(self, data):
 
     def test_setitem_scalar_key_sequence_raise(self, data):
         arr = data[:5].copy()
-        with pytest.raises(ValueError):
+        # complex128 data raises TypeError; other numeric types raise ValueError
```

Comment:
any idea why?  would it make sense to catch and re-raise as a ValueError for consistency?